src/world/scheduler/builder.rs

```diff
@@ -262,6 +262,29 @@ impl WorkloadBuilder {
             Err(err) => panic!("{:?}", err),
         }
     }
+    /// Extends the builder with the following [`WorkloadBuilder`].
```

Comment:
```suggestion
    /// Moves all systems of `other` into `Self`, leaving `other` empty.  
```

Same description as `Vec::append` https://doc.rust-lang.org/alloc/vec/struct.Vec.html#method.append
What do you think?